dist/sizzle.js

```diff
@@ -671,8 +671,10 @@ setDocument = Sizzle.setDocument = function( node ) {
 
 					// Fall back on getElementsByName
 					elems = context.getElementsByName( id );
+					length = elems.length;
 					i = 0;
```

Comment:
This could moved to the body of a `for` right?

Reply:
Yes, for both `i` and `length`. However, `i` was already outside of the `for`, so I decided to follow the existing style.

Reply:
Oh, that because of the https://contribute.jquery.org/style-guide/js/#good-examples, i.e. it should be fine to put `i` to the body, as long its initialised on top of the scope, like we doing here – https://github.com/jquery/jquery/blob/692f9d4db30c9c6c4f6bc76005cf153586202fa6/src/effects.js#L614 :)